StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1629UnitTests.cs

```diff
@@ -544,6 +544,23 @@ public interface ITest
             await VerifyCSharpFixAsync(testCode, expected, fixedTestCode, default).ConfigureAwait(false);
         }
 
+        [Theory]
+        [InlineData("a")]
+        [InlineData("see")]
+        [InlineData("seealso")]
```

Comment:
📝 `<seealso>` is a section-level element, so it would not be expected inside `<summary>`.